localstack/utils/container_utils/docker_cmd_client.py

```diff
@@ -478,9 +478,9 @@ def login(self, username: str, password: str, registry: Optional[str] = None) ->
 
     def has_docker(self) -> bool:
         try:
-            run(self._docker_cmd() + ["ps"])
+            run(self._docker_cmd() + ["ps"], timeout=5)
             return True
-        except (subprocess.CalledProcessError, FileNotFoundError):
+        except (subprocess.CalledProcessError, FileNotFoundError, TimeoutError):
```

Comment:
I could not find anywhere in the python docs, why a `TimeoutError` would be thrown, it should be a `TimeoutExpired`, which is a class derived from SubprocessError.
Did you verify this exception being thrown?

Reply:
Thanks for catching that - I had not realised it was a separate exception. T'was a hasty PR this morning but I'll sort this out and test :+1:

Reply:
Hmm the plot thickens... I cannot reproduce this issue now that I'm back.. I will probably close for now, but thanks for taking a look!